src/screens/Login/Login.js

```diff
@@ -61,7 +69,9 @@ const Login = props => {
       <View style={styles.loginContainer}>
         <View style={styles.headerContainer}>
           <View style={styles.infoImgContainer}>
-            <Image style={styles.infoImg} source={infoImg} />
+            <TouchableHighlight onPress={() => { navigation.navigate(FIRST_INTRO_SCREEN); }}>
```

Comment:
Your PR looks great! But could you please move onPress handler to dedicated function? For me it would enhance readibility

Reply:
done